src/vos/ilog.c

```diff
@@ -1497,7 +1451,9 @@ set_entry(struct ilog_entries *entries, const struct ilog_id *id, bool punch,
 }
 
 int
-ilog_fetch(daos_handle_t loh, uint32_t intent, struct ilog_entries *entries)
+ilog_fetch(struct umem_instance *umm, struct ilog_df *root_df,
```

Comment:
I'm a little curious, why ilog_fetch is different with other functions which require open?